### DISCUSSION TITLE
Receiving a 'FutureWarning' in Terminal

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1961] [Q&A]
Dear all, 

I recently switched the model in ASReview to Doc2Vec and Random Forest, and for this, I installed new packages. I used the _pip install asreview [all]_ command to do this. Ever since then, I have been receiving some weird messages in my Terminal/Command Prompt. Below is a copy of the current one, should I be worried that this affects the screening in ASReview?

_/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/asreview/io/utils.py:142: FutureWarning: A value is trying to be set on a copy of a DataFrame or Series through chained assignment using an inplace method.
The behavior will change in pandas 3.0. This inplace method will never work because the intermediate object on which we are setting values always behaves as a copy._

_For example, when doing 'df[col].method(value, inplace=True)', try using 'df.method({col: value}, inplace=True)' or df[col] = df[col].method(value) instead, to perform the operation inplace on the original object._

  _df[col].fillna(LABEL_NA, inplace=True)_

Many thanks in advance for your input!

**Comment by timovdk** [comment]
Dear @Morle2104,
This specific warning you are seeing is to warn that in version 3.0 of pandas (a Python package that ASReview uses) some behaviour will change. Since ASReview uses pandas version 2.2.3, this is not a problem for you currently.

I cannot say anything about the other warnings you are seeing, so if they are not of type 'FutureWarning' please post them here as well!
